src/ru/skypro/Book.java

```diff
@@ -0,0 +1,42 @@
+package ru.skypro;
+
+import java.util.Objects;
+
+public class Book {
+  private String name;
+  private int godPublic;
+  private Author author;
+   public Book(String name,int godPublic,Author author){
```

Comment:
Между полями и методами следует оставлять пустую строка в качестве отступа для лучшей читаемости кода
```suggestion
  private Author author;
  
   public Book(String name,int godPublic,Author author){
```
используйте ctrl+alt+L для автоформатирования кода